lib/utf8.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+module.exports.decode = function (array) {
+  return new Buffer(array).toString();
+};
+
+module.exports.encode = function (string) {
+  return Array.from(new Buffer(string));
```

Comment:
No, this is not good. This means doing another O(n) iteration over the buffer, in addition to the iteration being done over the string.